projects/packages/status/src/class-request.php

```diff
@@ -59,7 +61,7 @@ public static function is_frontend() {
 		 *
 		 * @param bool $is_frontend Whether the current request is for accessing the frontend.
 		 */
```

Comment:
Same with `@since $$next-version$$ Added arg` to reflect that it's not original.